src/sentry/static/sentry/app/views/performance/trends/chart.tsx

```diff
@@ -61,31 +60,35 @@ function transformEventStats(data: EventsStatsData, seriesName?: string): Series
   ];
 }
 
-function transformEventStatsSmoothed(data: Series[], seriesName?: string): Series[] {
+function transformEventStatsSmoothed(data: Series[], seriesName?: string) {
```

Comment:
Nit: Perhaps the type here should be `data: [Series]` as you're assuming it's a single series? Maybe even lift the series data out of the array before this function?

Reply:
I'm not too worried about that finely grained level of typing, I don't think it provides much value. If I was expecting to possibly return mixed values in the future I'd change it, and typing out 'tuples' I think should probably be reserved when you know there will be individual manipulation of the fields and not just a simple 'pick a single item' or a map.